python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -69,6 +67,16 @@ def test_library_creation_deletion(moto_s3_uri_incl_bucket):
         _lib = ac["pytest_test_lib"]
 
 
+def test_library_creation_deletion_s3(moto_s3_uri_incl_bucket):
+    ac = Arctic(moto_s3_uri_incl_bucket)
+    library_creation_deletion(ac)
+
+
+def test_library_creation_deletion_lmdb(tmpdir):
+    ac = Arctic(f"lmdb://{tmpdir}")
```

Comment:
Minor: can we create a trivial fixture that does just this so that we can do it properly centrally in the future?